Sources/OpenAPIRuntime/Conversion/Configuration.swift

```diff
@@ -96,6 +96,10 @@ extension JSONDecoder.DateDecodingStrategy {
     }
 }
 
+public protocol DecodingErrorHandler: Sendable {
+    func willThrow(_ error: any Error)
```

Comment:
Would you be interested in the context this was thrown in? See `ClientError` and `ServerError`, where we add more context.